ShopifySharp/Services/Fulfillment/FulfillmentService.cs

```diff
@@ -89,6 +89,27 @@ public virtual async Task<Fulfillment> CreateAsync(long orderId, Fulfillment ful
             return response.Result;
         }
 
+        /// <summary>
+        /// Creates a new <see cref="Fulfillment"/> on the order.
+        /// </summary>
+        /// <param name="orderId">The order id to which the fulfillments belong.</param>
+        /// <param name="fulfillment">A new <see cref="Fulfillment"/>. Id should be set to null.</param>
+        /// <param name="cancellationToken">Cancellation Token</param>
+        /// <returns>The new <see cref="Fulfillment"/>.</returns>
+        public virtual async Task<Fulfillment> CreateForFulfillmentAsync(FulfillmentShipping fulfillment, CancellationToken cancellationToken = default)
```

Comment:
Should this be named `CreateForOrderAsync`?

Reply:
In this case, no. The `FulfillmentShipping` object contains an array of `fulfillment_order_id` which are not the same as `order_id`. However, I did notice that `<param name="orderId">` should not be there as it is not relevant to this method.